src/Compilers/Test/Resources/Core/NetFX/ValueTuple/TupleElementNamesAttribute.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
I think I changed the license to match Roslyn when copying the VT code from corefx. Maybe do the same here.